zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -1119,19 +1119,19 @@ cfu.options.dialog.dirs.remove.button.confirm	= Remove
 cfu.options.dialog.dirs.remove.button.cancel	= Cancel
 cfu.options.dialog.dirs.remove.checkbox.label	= Do not show this message again
 cfu.options.dir.border 				= Directories
-cfu.options.downloadNewRelease 		= Automatically download new ZAP releases (you will be prompted to install them)
-cfu.options.downloadNewRelease.kali = Automatically download new ZAP releases (disabled on Kali)
-cfu.options.checkAddonUpdates		= Check for updates to the add-ons you have installed
-cfu.options.installAddonUpdates		= Automatically install updates to the add-ons you have installed
-cfu.options.installScannerRules		= Automatically install updates to the scanner rules you have installed
+cfu.options.downloadNewRelease 		= Automatically download new ZAP releases (you will be prompted to install them).
+cfu.options.downloadNewRelease.kali = Automatically download new ZAP releases (disabled on Kali).
+cfu.options.checkAddonUpdates		= Check for updates to the add-ons you have installed.
+cfu.options.installAddonUpdates		= Automatically install updates to the add-ons you have installed.
+cfu.options.installScannerRules		= Automatically install updates to the scan rules you have installed.
```

Comment:
In the installer it's using `scanner`:
https://github.com/zaproxy/zaproxy/blob/430d52792b7da1e57492f8deb21d667ff08b1f0c/zap/src/main/installer/resources/Messages.properties#L15

Reply:
Are we in agreement that "scan rule" is better? If so I'll tweak the installer set too.